Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM tomcat
```

Comment:
Maybe add some documentation for dev who wants to launch the image ?

Reply:
Done